sampletester/cli.py

```diff
@@ -55,7 +60,7 @@
 DEFAULT_LOG_LEVEL = 100
 
 # Set this to True to get a backtrace for debugging
-DEBUGME=False
+DEBUGME=True
```

Comment:
Does it make sense to make this a flag?

Reply:
Possibly, but I'm inclined to not do so because normal users should not be debugging, and the debug output is not very readable. But it is an oversight that I left it on!